src/deriveProof.ts

```diff
@@ -74,13 +107,22 @@ export const deriveProof = async (
     delete compactProof[alias];
     delete compactProof["@context"];
 
+    /**
```

Comment:
Can we create an issue to capture this and come back to it with a more permanent solution?

Reply:
Done: https://github.com/mattrglobal/jsonld-signatures-bbs/issues/85